addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -169,6 +169,7 @@
                                         <label for="contact_list_ids" string="Select mailing lists:" class="oe_edit_only"/>
                                         <field name="contact_list_ids" widget="many2many_tags"
                                             placeholder="Select mailing lists..." class="oe_inline"
+                                            context="{'form_view_ref': 'mass_mailing.mailing_list_view_form_simplified_m2m'}"
```

Comment:
Why you did not use `mailing_list_view_form_simplified` directly ?
The only difference is the footer buttons, how did you get stat buttons in the simplified form view ?

Reply:
Hi! Yes, that was the reason. As it is, the buttons are "save and new" / "save and close" / discard. So I just wanted to keep the present config. As it is a m2m, I thought this would be cleaner (hence the new view :) ).
I guessed that by removing the footer, it created the default creation footer for m2m fields, so I removed the footer of the existing view in the heriting new one.

Also, I tried again without the new view and realised "create" and "discard" buttons were not closing the view when using mailing_list_view_form_simplified in create and edit as per spec. I do not know much about the special="cancel" but the name="close_dialog" refers to an action on the mailing.list model. I must have missed something? Any hint? :)

Thanks!

Reply:
@Odoonan Hum, ok I see, and if you remove the footer on the original simplified view it will not be closed automatically in the mailing list kanban view :/

So maybe can you just "inverse" and remover the footer on the simplified view and add the footer with xpath in a new view  `mailing_list_view_form_simplified_footer`, I see it's how we did in other modules (project...)

Reply:
@std-odoo Indeed, this make sense to do so. Not much work is required since the view is not widely used. I did the change and all seems to work on Runbot.

Cheers